configs/envoy_double_proxy_v2.template.yaml

```diff
@@ -8,17 +8,20 @@
   filter_chains:
   - filter_chain_match: {}
     {% if tls %}
-    tls_context:
-      common_tls_context:
-        tls_certificates:
-        - certificate_chain:
-            filename: certs/servercert.pem
-          private_key:
-            filename: certs/serverkey.pem
-        validation_context: {}
-        alpn_protocols:
-        - h2
-        - http/1.1
+    transport_socket:
+      name: tls
```

Comment:
Should we create an alias `envoy.tls` and use this everywhere?

Reply:
Yeah I thought about that, do double registration is possible but to do it properly (with deprecation) will result another large PR. We might want to do that sometime anyway but let's keep this for now? `tls` transport socket is there for quite a while.

Reply:
Fair enough, can you file an issue?

Reply:
https://github.com/envoyproxy/envoy/issues/8540